src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -248,12 +256,22 @@ public static ReportNode createLoadFlowReporter(ReportNode reportNode, String ne
                 .add();
     }
 
-    public static ReportNode createLfNetworkReporter(ReportNode reportNode, int networkNumCc, int networkNumSc) {
-        return reportNode.newReportNode()
+    public static ReportNode createRootLfNetworkReportNode(int networkNumCc, int networkNumSc) {
+        return ReportNode.newRootReportNode()
+                .withMessageTemplate("lfNetwork", "Network CC${networkNumCc} SC${networkNumSc}")
+                .withUntypedValue(NETWORK_NUM_CC, networkNumCc)
+                .withUntypedValue(NETWORK_NUM_SC, networkNumSc)
+                .build();
+    }
+
+    public static ReportNode createLfNetworkReportNode(ReportNode reportNode, ReportNode lfNetworkReportNode, int networkNumCc, int networkNumSc) {
+        ReportNode newReportNode = reportNode.newReportNode()
                 .withMessageTemplate("lfNetwork", "Network CC${networkNumCc} SC${networkNumSc}")
                 .withUntypedValue(NETWORK_NUM_CC, networkNumCc)
                 .withUntypedValue(NETWORK_NUM_SC, networkNumSc)
                 .add();
+        newReportNode.include(lfNetworkReportNode);
+        return newReportNode;
```

Comment:
this will need to be revisited at next powsybl-core bump because of https://github.com/powsybl/powsybl-core/pull/3007

Reply:
This is done in #1034.